lib/host/aws.js

```diff
@@ -20,11 +20,13 @@ AWS Metadata service endpoint.
 @const
 @see http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AESDG-chapter-instancedata.html
 */
-var BASE_URL = 'http://169.254.169.254/2012-01-12';
+var BASE_URL = 'http://169.254.169.254/latest';
```

Comment:
Is this deliberate? I suspect you just fixed it to `latest` inorder to get the spot shutdown notifications.
Note, you can probably list available versions with `curl http://169.254.169.254/` on ec2.

We can also just hope aws maintains backwards compatibility, as they have so far. We're not going to see breakage when they add more keys to the meta-data service.